.github/workflows/verify-ec2.yml

```diff
@@ -2,9 +2,9 @@ name: EC2 Testing Matrix
 
 on:
   push:
-    branches: [ master ]
```

Comment:
Is the master branch going to be deleted? Will the main branch get switched to default? What's the plan for the branches after merging this?